releases/release-4.0.8.md

```diff
@@ -0,0 +1,159 @@
+---
+title: TiDB 4.0.8 Release Notes
+---
+
+# TiDB 4.0.8 Release Notes
+
+Release date: October 29, 2020
+
+TiDB version: 4.0.8
+
+## New Features
+
++ TiDB
+
+    - Support new aggregate function APPROX_PERCENTILE [#20197](https://github.com/pingcap/tidb/pull/20197)
+
++ TiFlash
+
+    - Support pushing down `CAST` functions
+
++ Tools
+
+    + TiCDC
+
+        - Support snapshot-level consistency replication [#932](https://github.com/pingcap/ticdc/pull/932)
+
+## Improvements
+
++ TiDB
+
+    - Prioritize low selectivity indexes in the greedy search procedure of `Selectivity()` [#20154](https://github.com/pingcap/tidb/pull/20154)
+    - Record more RPC runtime information in cop runtime stats [#19264](https://github.com/pingcap/tidb/pull/19264)
+    - Introduce errors documentation generator to generate errors.toml [#20564](https://github.com/pingcap/tidb/pull/20564)
+    - Speed up parse slow-log when query slow_query. [#20556](https://github.com/pingcap/tidb/pull/20556)
+    - When verifying potential new plans, plan binding will now wait before timing out worse plans so that more debug information can be written to the TiDB error log. [#20530](https://github.com/pingcap/tidb/pull/20530)
+    - Add execution retry time in slow log and slow_query. [#20495](https://github.com/pingcap/tidb/pull/20495)
+    - Add execution retry count in slow log and slow_query. [#20494](https://github.com/pingcap/tidb/pull/20494)
+    - Add system table `table_storage_stats` [#20431](https://github.com/pingcap/tidb/pull/20431)
+    - Add rpc runtime stats information for insert/update/replace statement. [#20430](https://github.com/pingcap/tidb/pull/20430)
+    - Add executor information in `explain for connection` result. [#20384](https://github.com/pingcap/tidb/pull/20384)
+    - Config: check if valid storage by simple indexing [#20342](https://github.com/pingcap/tidb/pull/20342)
```

Comment:
We don't need this PR in the release note, it's not related to the end-user at all.

Reply:
Removed in https://github.com/pingcap/docs/pull/4087/commits/19ef004986a496e2b6616d2185782cb04dbe1b4f